package.json

```diff
@@ -14,6 +17,7 @@
     "react-dom": "^16.13.1",
     "react-redux": "^7.2.0",
     "react-scripts": "3.4.1",
+    "react-testing-library": "^8.0.1",
```

Comment:
Yay - testing framework. This looks like it got moved. [@testing-library/react](https://www.npmjs.com/package/react-testing-library)